src/main/java/de/doubleslash/keeptime/view/ViewController.java

```diff
@@ -505,6 +504,8 @@ private GridPane setUpAddNewProjectGridPane(final Dialog<Project> dialog) {
 
       final ManageProjectController manageProjectController = loader.getController();
 
+      dialog.getDialogPane().lookupButton(ButtonType.OK).disableProperty().bind(manageProjectController.formValidProperty().not());

```

Comment:
when editing a project, the "hint" to set a name also shows. but the button is not disabled.
should disable it also while editing the project. maybe we can reuse the code somehow?

Reply:
Button is now disabled